### DISCUSSION TITLE
How to create my own dialog like ColumnPickerDialog

### DISCUSSION THREAD

**Discussion by pradeep-pandit** [serenity-is/Serenity#6298] [Help]
I have one requirement to move data from one side to another and vice a versa.

![image](https://user-images.githubusercontent.com/94299697/156215361-b7013f35-9b40-4ac9-bbcb-69a120f7ed14.png)

When I click on (+) from unconfigured bill head then it should display in left side and vice a versa.

Thanks in advance.

**Comment by stixoffire** [comment]
Pseudo code - look at how the checkbox grid is handle that will show you how to reference the row and get the item .. then simple add to one and remove from the other ..
`clickHandler(e) { e.itemClicked;   var myItem = getItems()[p.row]  configBillH.view.addItem(myItem) uconfigBillH.view.removeItem(myItem)   }
configBillH.render() uconfigBillH.render()  `
